plugins/lang/admincmd.txt

```diff
@@ -120,35 +120,35 @@ PLUGINS_RUN = %d Plugins, %d Plugins laufen
 LOADED_MODULES = Momentan geladene Module
 NUM_MODULES = %d Module
 FILE_NOT_FOUND = Datei "%s" nicht gefunden
-ADMIN_CONF_1 = ADMIN: fuehrt Config %s aus
-ADMIN_CONF_2 = ADMIN %s: fuehrt Config %s aus
+ADMIN_CONF_1 = ADMIN: führt Config %s aus
+ADMIN_CONF_2 = ADMIN %s: führt Config %s aus
 PAUSED = pausiert
 UNPAUSED = Pause beendet
-UNABLE_PAUSE = Server konnte das Spiel nicht anhalten. Reale Spieler werden auf dem Server benoetigt.
-SERVER_PROC = Server fuehrt %s aus
+UNABLE_PAUSE = Server konnte das Spiel nicht anhalten. Reale Spieler werden auf dem Server benötigt.
+SERVER_PROC = Server führt %s aus
 PAUSING = Pausieren
 UNPAUSING = Pause beendet
-PAUSE = Pause
-UNPAUSE = Pause beendet
+PAUSE = pausiert
+UNPAUSE = setzt fort
```

Comment:
Okay, I'll leave/change it - even though I think it should only be used by admincmd.sma because of "admincmd.txt"
I agree that there should be a rewrite to admincmd.sma....

Reply:
Done